checks/microbenchmarks/cpu_gpu/node_burn/node-burn-ce.py

```diff
@@ -97,12 +109,13 @@ def setup_job(self):
 
         # On GH200 use 1 task per GH module
         if proc.arch == 'neoverse_v2':
-            self.num_tasks = self.num_sockets
+            self.num_tasks_per_node = self.num_sockets
             self.num_cpus_per_task = self.cpus_per_socket
         else:
-            self.num_tasks = 1
+            self.num_tasks_per_node = 1
             self.num_cpus_per_task = self.cpus_per_socket * self.num_sockets
```

Comment:
is OMP_NUM_THREADS needed ?

Reply:
I uses all the cores available according to the number of cpus per task